src/core/api.ts

```diff
@@ -25,9 +25,12 @@ export interface GoslingApi {
     zoomToExtent(viewId: string, duration?: number): void;
     zoomToGene(viewId: string, gene: string, padding?: number, duration?: number): void;
     suggestGene(viewId: string, keyword: string, callback: (suggestions: GeneSuggestion[]) => void): void;
+    getTracksAndViews(): VisUnitApiData[];
     getViewIds(): string[];
```

Comment:
Do we want to keep this API method? It could be confusing because `getViewIds` returns the `view.uid` created by HiGlass, not `view.id` as the name would suggest.

Reply:
Ah, good catch! I tried to change the name to `getTrackIds` in another PR (https://github.com/gosling-lang/gosling.js/pull/914) to make any HiGlass concept hidden from the users. I will rename it in this PR.

Reply:
Actually, let me change that in another PR. I will work on a PR to address https://github.com/gosling-lang/gosling.js/issues/909. In the PR, I will try to make the higlass concept hidden from users in the APIs (like the HiGlass view IDs) and better manage the connection between Gosling track IDs and HiGlas view IDs.